scripts/test/test-with-eksctl.sh

```diff
@@ -158,16 +158,15 @@ function run_integration_test() {
   --aws-region=$AWS_REGION \
   --aws-vpc-id=$VPC_ID) || TEST_FAILED=true
 
+  # SGP + Metrics for Regression Test
   if [[ $additional_gingko_params == "--focus=LOCAL" ]]; then
-    # SGP + Metrics for Regression Test
-    ng=$(aws eks list-nodegroups --cluster-name $CLUSTER_NAME --query 'nodegroups[0]' --output text)
-    asg_name=$(aws eks describe-nodegroup --nodegroup-name $ng --cluster-name $CLUSTER_NAME --query 'nodegroup.resources.autoScalingGroups[?amiType=='AL2_x86_64'].name' --output text)
-    asg_config=$(aws eks describe-nodegroup --nodegroup-name $ng --cluster-name $CLUSTER_NAME --query 'nodegroup.scalingConfig')
-    max_size=$(echo $asg_config | jq '.maxSize')
-    min_size=$(echo $asg_config | jq '.minSize')
-    desired_size=$(echo $asg_config | jq '.desiredSize')
     # We need a larger group of nodes to test the regression of controllers
-    aws autoscaling update-auto-scaling-group --auto-scaling-group-name $asg_name --max-size=150 --desired-capacity=100 --min-size=0
```

Comment:
This does fix the issue but it also means that we cannot use it against beta cluster as eksctl isn't supported. A suggestion lets skip this test if its beta and follow it up with another PR to make it work against beta clusters